pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -172,6 +172,21 @@ func (s *apiScaffolder) scafffoldControllerWithImage(scaffold *machinery.Scaffol
 			return fmt.Errorf("error scaffolding container port in the controller: %v", err)
 		}
 	}
+
+	if len(s.runAsUser) > 0 {
```

Comment:
Also, we need to update the e2e tests for the Memcached and begin to pass the flag with the value